### ISSUE TITLE
Redundancy: Comparison

### ISSUE THREAD

**Issue by flipdazed** [flipdazed/SoftwareDevelopment#54]
**Bugged Version** 0_0_0
**Lines** `239; 241; 258; 271`

**Issue** 
This comparison is redundant as it has already been implicitly set to `True` in both cases.

```
                    # The card can't be in the list if PC can't afford to buy it
                    if money >= central['active'][int(source)].cost:
                        ...
                    else:
                        print "Error Occurred"
                    ...
                    # The card can't be in the list if PC can't afford to buy it
                    if money >= central['supplement'][0].cost:
                        ...
                    else:
                        print "Error Occurred"
```

The comparison has already been made on line `239`,

```
            # Primary purchase choice: Buy one supplement
            if len(central['supplement']) > 0:                  # If there are any supplements
                if central['supplement'][0].cost <= money:      # If PC has enough money
                    # Add to temporary purchases
                    templist.append(("S", central['supplement'][0]))
```

and on line `241`,

```
            # Secondary purchase choice: Buy supplements if available
            for intindex in xrange (0, central['activesize']):  # Loop all cards
                if central['active'][intindex].cost <= money:   # if PC has enough money
                    # Add to temporary purchases
                    templist.append((intindex, central['active'][intindex]))
```

**Fix**
Take out comparison.

**Comment by flipdazed** [comment]
upgraded as this will remove a confusing loop

**Comment by flipdazed** [comment]
it may be worth leaving this in if it can lead to a shared function for buying between the player and the computer

**Comment by flipdazed** [comment]
related #92 (closed for future development)

**Comment by flipdazed** [comment]
Removed as part of the fixed with #114 
changed to the following in both instances.
Left in to notify the developer of future errors that may be introduced.

```
            # This is kept here to avoid future errors that may be introduced
            # This is already verified as never being initiated earlier in the code
            # see can_afford_cards assignment
            self.logger.debug("Developer Error: There are no supplements to buy! {}")
            self.logger.debug("Money: {}".format(self.money))
            return
```

**Comment by flipdazed** [comment]
Also tested. Closed.
